flink-table/flink-table-api-java/src/test/java/org/apache/flink/table/expressions/resolver/ExpressionResolverTest.java

```diff
@@ -98,7 +99,7 @@
                                                         DataTypes.FIELD("n0", DataTypes.BIGINT()),
                                                         DataTypes.FIELD("n1", DataTypes.STRING())))
                                         .build())
-                        .select($("f0").flatten())
+                        .select(col("f0").flatten())
```

Comment:
Does it maybe make sense to have a dedicated test? 1st to be 100% visible, 2nd to avoid a future change that will make us loose that single test.